backend/src/main/java/atlantbh/restaurants/controllers/MenuController.java

```diff
@@ -0,0 +1,19 @@
+package atlantbh.restaurants.controllers;
+
+import atlantbh.restaurants.models.Menu;
+import atlantbh.restaurants.models.filters.MenuFilterBuilder;
+import atlantbh.restaurants.services.MenuService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class MenuController extends BaseController<Menu, MenuService> {
+
+    public ResponseEntity filter(@RequestParam Long restaurantId) {
+
+        MenuFilterBuilder mfb = new MenuFilterBuilder()
```

Comment:
watch out on pagination, if you have more than 10 items in result, they won't be displayed. You can pass 0 to pageSize from frontend to get all results.